apis/v1alpha3/types.go

```diff
@@ -340,6 +340,9 @@ type VirtualMachine struct {
 
 	// Network is the status of the VM's network devices.
 	Network []NetworkStatus `json:"network"`
+
+	// VMRef is the the VM's Managed Object Reference on vSphere.
+	VMRef string `json:"vmRef"`
```

Comment:
VirtualMachine seems to be spelled out elsewhere. We might want to stick with that pattern for consistency.

Reply:
VMRef is an internal concept of vSphere :) This is why we are using this naming